.travis.yml

```diff
@@ -1,6 +1,18 @@
-language: c
+language: python
+
+python:
+- '3.6'
+
+services:
+- mongodb
+- docker
 
 before_install:
 - sudo apt-get install libcunit1 libcunit1-dev
+- pip3 install git+https://github.com/Maxmawt/INGInious # Until UCL-INGI/INGInious has fixed his part ;-)
```

Comment:
Is this still needed ? Could you refer to a specific issue we should track on the INGInious repository ?

Reply:
It was #414, but I don't know if it has been fixed since then. I'll check.